<ui version="4.0" >
 <class>QgsOGRSourceSelect</class>
 <widget class="QDialog" name="QgsOGRSourceSelect" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>433</width>
    <height>454</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>OGR Sources</string>
  </property>
  <property name="windowIcon" >
   <iconset/>
  </property>
  <property name="sizeGripEnabled" >
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="layoutWidget" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>420</y>
     <width>415</width>
     <height>29</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" >
    <property name="margin" >
     <number>0</number>
    </property>
    <property name="spacing" >
     <number>6</number>
    </property>
    <item>
     <widget class="QPushButton" name="btnHelp" >
      <property name="enabled" >
       <bool>true</bool>
      </property>
      <property name="text" >
       <string>Help</string>
      </property>
      <property name="shortcut" >
       <string>F1</string>
      </property>
      <property name="autoDefault" >
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <spacer>
      <property name="orientation" >
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType" >
       <enum>QSizePolicy::Expanding</enum>
      </property>
      <property name="sizeHint" >
       <size>
        <width>140</width>
        <height>27</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="btnAdd" >
      <property name="text" >
       <string>Add</string>
      </property>
      <property name="shortcut" >
       <string/>
      </property>
      <property name="autoDefault" >
       <bool>true</bool>
      </property>
      <property name="default" >
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnCancel" >
      <property name="text" >
       <string>Close</string>
      </property>
      <property name="shortcut" >
       <string/>
      </property>
      <property name="autoDefault" >
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox" >
   <property name="geometry" >
    <rect>
     <x>9</x>
     <y>9</y>
     <width>415</width>
     <height>97</height>
    </rect>
   </property>
   <property name="title" >
    <string>OGR Connections</string>
   </property>
   <layout class="QGridLayout" >
    <property name="margin" >
     <number>9</number>
    </property>
    <property name="spacing" >
     <number>6</number>
    </property>
    <item row="1" column="3" >
     <widget class="QPushButton" name="btnDelete" >
      <property name="text" >
       <string>Delete</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2" >
     <widget class="QPushButton" name="btnEdit" >
      <property name="text" >
       <string>Edit</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1" >
     <widget class="QPushButton" name="btnNew" >
      <property name="text" >
       <string>New</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" >
     <widget class="QPushButton" name="btnConnect" >
      <property name="text" >
       <string>Connect</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="4" >
     <widget class="QComboBox" name="cmbConnections" />
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_2" >
   <property name="geometry" >
    <rect>
     <x>9</x>
     <y>112</y>
     <width>415</width>
     <height>305</height>
    </rect>
   </property>
   <property name="title" >
    <string>Tables:</string>
   </property>
   <layout class="QGridLayout" >
    <property name="margin" >
     <number>9</number>
    </property>
    <property name="spacing" >
     <number>6</number>
    </property>
    <item row="0" column="0" >
     <widget class="QListWidget" name="lstTables" />
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <tabstops>
  <tabstop>cmbConnections</tabstop>
  <tabstop>btnConnect</tabstop>
  <tabstop>btnNew</tabstop>
  <tabstop>btnEdit</tabstop>
  <tabstop>btnDelete</tabstop>
  <tabstop>btnHelp</tabstop>
  <tabstop>btnAdd</tabstop>
  <tabstop>btnCancel</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>btnCancel</sender>
   <signal>clicked()</signal>
   <receiver>QgsOGRSourceSelect</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>404</x>
     <y>446</y>
    </hint>
    <hint type="destinationlabel" >
     <x>229</x>
     <y>236</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
